languages/en.ini

```diff
@@ -1092,6 +1094,7 @@ relais_success_message = "Request id #%%id%% was created. You will receive a con
 Related Author = "Related Author"
 Related Items = "Related Items"
 Related Subjects = "Related Subjects"
+remove_selected_filters = "Remove selected filters"
```

Comment:
Is this wording potentially confusing because it implies that you can select multiple filters and then remove them in a single operation? Would it be better for the heading to be something more like "Currently applied filters" and then have each individual link say "remove [filter]"?

Reply:
We already have some sr-only text inside the removable individual filters (just the translation was missing), so I added that and added the plural for the Applied filters string.

Reply:
Similar to the above, maybe it's worth renaming the new string to applied_filters_label and including the colon in the string itself.

(And I do realize that VuFind has some existing situations where the colons are applied in the templates instead of in the language files, so there is a precedent for doing it the way it's done here... I just think internationalizing the punctuation is likely to be more flexible in the long run, and if we decide to change the other usages in the future, it will be easier if we don't add new instances).

Reply:
this is gone completely from the code as it is no longer necessary